docs/DeveloperGuide.md

```diff
@@ -406,9 +406,9 @@ file with the modified workout list whenever a workout is deleted in the applica
 <br><br>
 An alternative considered was to find the workout to be deleted in the resource file, and then
 remove that workout. This is a faster implementation, however, more complicated due to the way
-the workouts are formatted and stored in the `workouts.txt` file.
+the workout data are formatted and stored in the `workouts.txt` file.
 <br><br>
-Hence, to simplify the coding part, the team decided to implement our current implementation - that
+Hence, to simplify the implementation, the team decided to implement our current implementation - that
 is, to rewrite all workout to the resource file whenever a workout is deleted.
```

Comment:
Perhaps the sentence can be more concise:

> Hence, to simplify the implementation, the team decided to ~implement our current implementation - that
is, to~ **simply** rewrite all **workouts** to the resource file whenever a workout is deleted.

Note: I bolded the text to highlight the edited words, but they should not be bolded in the actual change 😬